### ISSUE TITLE
S1450 detects false positive with fields.

### ISSUE THREAD

**Issue by Portikus** [SonarSource/sonar-csharp#229]
Hello,

I have the following code:

```csharp
public class Class1
{
    private int field;
    public int AnswerToAllQuestions()
    {
        return this.field;
    }
    public void SetAnswer(int answer)
    {
        this.field = answer;
    }
}
```

The analyser gives me the hint that I am able to remove the "field" field. S1450.
But this is impossible. In my code I get over 500 of these messages.

**Comment by fmallet** [comment]
Relates to #158

**Comment by Evangelink** [comment]
Hi @Portikus ! 
Could you tell us if you are experiencing this issue with SonarLint for Visual Studio or with the Scanner for MSBuild? If it is with SonarLint, which version of Visual Studio are you using?

**Comment by Portikus** [comment]
Hi @Evangelink ,
I am using the SonarAnalyser.Csharp Roslyn Analyser nuget package version 5.9.0.1001. And yes I am using Visual Studio 2017 version 15.1 (26403.7)

**Comment by Evangelink** [comment]
Hi @Portikus .
I will close this issue then and if you still experience the issue with the next release please feel free to open it again or create another one.

Cheers,
Amaury
